apps/dashPOC/rollup.config.js

```diff
@@ -42,8 +41,9 @@ export default {
         postcss({
             extensions: [".css"],
             minimize: true
-        })/*,
-        uglify()
-        */
+        })
+        /*,
+                uglify()
+                */
```

Comment:
Is `uglify` a way of minimizing/obfuscating a rollup?

Reply:
Yes - but we normally use terser these days.